src/Components/Authorization/src/AuthorizeRouteView.cs

```diff
@@ -49,22 +49,22 @@ public AuthorizeRouteView()
         /// The content that will be displayed if the user is not authorized.
         /// </summary>
         [Parameter]
-        public RenderFragment<AuthenticationState> NotAuthorized { get; set; }
+        public RenderFragment<AuthenticationState>? NotAuthorized { get; set; }
 
         /// <summary>
         /// The content that will be displayed while asynchronous authorization is in progress.
         /// </summary>
         [Parameter]
-        public RenderFragment Authorizing { get; set; }
+        public RenderFragment? Authorizing { get; set; }
 
         /// <summary>
         /// The resource to which access is being controlled.
         /// </summary>
         [Parameter]
-        public object Resource { get; set; }
+        public object? Resource { get; set; }
```

Comment:
Do we need to set a default value for this in

https://github.com/dotnet/aspnetcore/blob/47686d16b4b644d18c848005c8e26fea7e7544fd/src/Components/Authorization/src/AuthorizeRouteView.cs#L94

Reply:
Not sure I follow. It should pass a null-value thru and AuthorizeAsync allows a null resource.

Reply:
Yeah, nevermind. `AuthorizeViewCore` is already updated to have a nullable `Resource` parameter.